Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ghcr.io/celestiaorg/celestia-app:v1.0.0-rc14 AS celestia-app
 
-FROM ghcr.io/celestiaorg/celestia-node:v0.11.0-rc12
+FROM ghcr.io/celestiaorg/celestia-node:v0.11.0-rc13
```

Comment:
```suggestion
FROM vietcelestia/celestia-node:v0.11.0-rc13
```

Reply:
hm. should this be `ghcr.io/vietcelestia/`

Reply:
nope. I'm using dockerhub 😅

Reply:
got it! should we use this or maybe wait until a compatible release is made?

Reply:
let's use this in the meantime and get back to compat release once we fix the pipelines